client/.storybook/daylightTheme.js

```diff
@@ -0,0 +1,20 @@
+import { create } from '@storybook/theming/create';
+
+export default create({
+  base: 'light',
+
+  textPrimaryColor: '#4D4C4C',
+  textSecondaryColor: '#707070',
+  textThirdColor: '#FFFFFF',
+  backgroundPrimaryColor: '#00A8CC',
+  buttonPrimaryColor: '#45BFD9',
+  warningColor: '#FEA630',
+  lightColor: '#FFBE68',
+  inputColor: '#EFEAEA',
+
+  colorPrimary: 'hotpink',
+  colorSecondary: 'deepskyblue',
+
+  brandTitle: 'My famly storybook',
+  brandUrl: 'https://github.com/meloneminze/famly',
```

Comment:
is this required for the `create()` method?

Reply:
It is in the exapmle. Do you think I can delete it?

Reply:
I will delete it. I had taken it over from the example.